src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -422,4 +445,35 @@ function AppleHealthMenuItem(props: AppleHealthMenuItemProps) {
             </div>
         }
     </div>;
+}
+
+function WeatherMenuItem(props: { settings: DataCollectionSettings | null, participantInfo: ParticipantInfo | null, postalCodeSurveyName: string }){
```

Comment:
I don't feel that strongly about it, but you can probably drop the `| null`'s from the types here.  I think those will need to be set by the time we get to this point, so the calling code could just use `!`.